.gitignore

```diff
@@ -3,6 +3,10 @@
 /deps
 erl_crash.dump
 *.ez
+mix.lock
```

Comment:
Unsure that this should be here. Personally, I would say "leave it", but http://elixir-lang.org/getting-started/mix-otp/dependencies-and-umbrella-apps.html says:

> The lock file must be checked in to your version control system, to guarantee that everyone who uses the project will use the same dependency versions as you.